src/Messages.js

```diff
@@ -0,0 +1,18 @@
+const GAME_MSG = Object.freeze({
+  START: "숫자 야구 게임을 시작합니다.",
+  END: "3개의 숫자를 모두 맞히셨습니다! 게임 종료",
+  RESTART: "게임을 새로 시작하려면 1, 종료하려면 2를 입력하세요.\n",
+  INPUT: "숫자를 입력해주세요 : ",
+  BALL: "볼",
+  STRIKE: "스트라이크",
+  NOTHING: "낫싱",
+});
+
+const ERROR_MSG = Object.freeze({
+  INPUT_ERROR_LEN: "[ERROR]세자리 수를 입력해주세요.",
+  INPUT_ERROR_DUPLE: "[ERROR]숫자가 중복되지 않아야 합니다.",
+  INPUT_ERROR_NOT_NUM: "[ERROR]숫자만 입력해주세요.",
+  RESTART_ERROR_NOT_ANS: "[ERROR]1 혹은 2를 입력해주세요.",
+});
```

Comment:
메시지들을 상수화 해서 사용하니깐 가독성이 높아져서 보기 좋은것같습니다😀
다만, 변수 이름을 작성할 때는 변수이름이 조금 길어지더라도 모두 작성하는게 어떨까..? 싶은 생각입니다.
(실제로 줄임말을 사용하지 않는게 추천하는 네이밍 컨벤션입니다!)
변수가 약간 길어지더라도 다른 사람들이 볼 때 훨씬 알아보기 쉬울것 같습니다!
```suggestion
  INPUT_ERROR_LENGTH: "[ERROR]세자리 수를 입력해주세요.",
  INPUT_ERROR_DUPLICATE: "[ERROR]숫자가 중복되지 않아야 합니다.",
  INPUT_ERROR_NOT_NUMBER: "[ERROR]숫자만 입력해주세요.",
  RESTART_ERROR_NOT_ANSWER: "[ERROR]1 혹은 2를 입력해주세요.",
});
```

Reply:
다른 사람들이 볼 때 알아보기 쉽게 작성하기..
아무 생각없이 너무 길면 안된다고 생각했는데 꼭 그런 것만은 아니군요! 참고하겠습니다 :) 감사합니다.